functions/gameManager.ts

```diff
@@ -0,0 +1,147 @@
+const AWS = require('aws-sdk');
+const isTest = process.env.JEST_WORKER_ID;
+const config = {
+  convertEmptyValues: true,
+  ...(isTest && {
+    endpoint: 'localhost:8000',
+    sslEnabled: false,
+    region: 'local-env',
+  }),
+};
+const docClient = new AWS.DynamoDB.DocumentClient(config);
+const table = (isTest) ? process.env.DYNAMO_DB_TEST_TABLE : process.env.DYNAMO_DB_GAME_TABLE;
+import { Game } from "./game";
+import { GameError } from "./gameErrorHandler";
+ 
+ export async function getGame(game: Game) : Promise<Game> {
+    let params = {
+      TableName: table,
+      Key: {
+        userID: game.userID,
+        gameName: game.gameName
+      },
+      KeyConditionExpression: `gameName = ${game.gameName} AND userID = ${game.userID}`
+    }
+    
+    try {
+      let response = await docClient.get(params).promise();
+      let game = serializeDynamoResponse(response.Item);
+      return game;
+    } catch (err: any) {
+      //Dynamo returns an empty object if the get can't find a record.
+      //Not sure how to handle this since the documentClient doesn't throw an error
+      if (err.message == "Cannot read property 'userID' of undefined") {
+        throw new GameError("Unable to find game.", 404);
+      }
+      throw new GameError(err.message, err.statusCode)
```

Comment:
probably can just remove this and let any other exception get raised normally. Should do this for all the GameManager methods.